Formula/fastuniq.rb

```diff
@@ -0,0 +1,19 @@
+class Fastuniq < Formula
+  # cite Xu_2012: "https://doi.org/10.1371%2Fjournal.pone.0052249"
```

Comment:
I believe `%2F` can usually be changed to `/`, which is more readable. `brew cite --ruby` could be patched to make that substitution.

Reply:
I didn't notice it, sorry..  i pasted it from PLOS/./

Reply:
No worries. It's a small thing, and `%2F` does work.